doc/source/cluster/key-concepts.rst

```diff
@@ -60,19 +60,31 @@ sit idle, the autoscaler will remove worker nodes from the cluster.
 It is important to understand that the autoscaler only reacts to task and actor resource requests, and not application metrics or physical resource utilization.
 To learn more about autoscaling, refer to the user guides for Ray clusters on :ref:`VMs <cloud-vm-index>` and :ref:`Kubernetes <kuberay-index>`.
 
+
+.. _ray_driver:
+
+Ray Driver
+----------
+
+A Ray driver is the process that calls the Ray tasks and actors in the user application.
+A driver is started by running `ray.init()`. The script that calls `ray.init()` may also be referred to as the "driver script".
+
+
 .. _cluster-clients-and-jobs:
 
-Ray Jobs
---------
+Ray Job
+-------
+
+A Ray Job is a user application that includes a Ray driver. There are two ways of running Ray Jobs on Ray clusters:
 
-The main method for running a workload on a Ray cluster is to use Ray Jobs.
-Ray Jobs enable users to submit locally developed-and-tested applications to a
+The recommended method for running a workload on a Ray cluster is to use Ray Job Submission.
+Ray Job Submission enable users to submit locally developed-and-tested applications to a
```

Comment:
I probably wouldn't capitalize Submission (lowercasing "jobs" also seems better but we can leave it for later).

```suggestion
Ray Job submission enables users to submit locally developed and tested applications to a
```